src/components/Hed/Hed.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Hed.scss';
+
+function Hed ( {
+	children,
+	size,
+} ) {
+	return (
+		<div className={`${styles.container} ${styles[ size ]}`}>
+			{children}
+		</div>
```

Comment:
Suggest using `span` so we can legitimately nest this component in a heading tag, for example. You'll need to add `display: block` too

```suggestion
		<span className={`${styles.container} ${styles[ size ]}`}>
			{children}
		</span>
```

Reply:
Meaning you want the ability to do this?

```
<h3><Hed>Get your flu shot as early as possible this year</Hed></h3>
```

I don't know why exactly, but assumed that for components like this, the semantic tag would be controlled directly or indirectly by a prop. I also notice that outside of the article h1 and a few rare exceptions, we don't use `h*` tags for heads. Regardless, I've seen a few UI libraries implement this as:

```
<Hed as="h1">Get your flu shot as early as possible this year</Hed>
```

Or via a `type` prop that is inspected by logic internal to `Hed`:

```
function Hed ( { children, type } ) {
  if ( 'canonical' === type ) {
    return <h1>{children}</h1>;
  }
  // ...
}
```

Mostly I'm tempted by the idea of making semantic tag selection something the consumer has to think about when passing props to our components. Instead of remembering to wrap `Hed` in an `h3`, they can inspect the `as` prop. But again, I'm not sure we actually do this in practice, except in cases like the article `h1` that probably deserve their own component.

Reply:
I think I'm coming around. I'm going to merge this and fix it up in the next PR.